src/UIKit/UIControl.cs

```diff
@@ -22,6 +22,11 @@ class UIControlEventProxy : NSObject {
 		UIControl source;
 		internal int Counter = 1;
 		internal const string BridgeSelector = "BridgeSelector";
+		internal static byte [] BridgeSelector8 = new byte [] {
+			(byte)'B', (byte)'r', (byte)'i', (byte)'d', (byte)'g', (byte)'e', (byte)'S', (byte)'e',
```

Comment:
C# 11 has UTF8 string literals:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/utf8-string-literals

But the docs also details a way a different way to do that (that uses embedded assembly metadata).

Reply:
Seems like that example is outdated.

https://devblogs.microsoft.com/dotnet/csharp-11-preview-updates/#utf-8-string-literals

Reply:
Right, can't wait to get access to that. In the meantime, it's byte arrays.